osu-wine

```diff
@@ -173,7 +173,7 @@ case "$1" in
         Info "Your PC isn't really a Steam Deck, no need to use this c.c" && exit 1
     else
         Info "Installing needed packages for Steam Deck.."
-        sudo pacman -Sy libxcomposite lib32-libxcomposite gnutls lib32-gnutls wine winetricks || Error "Check your connection or make sure you disabled read-only file system (read more at GitHub)"
+        sudo pacman -Sy libxcomposite lib32-libxcomposite gnutls lib32-gnutls wine-staging winetricks || Error "Check your connection or make sure you disabled read-only file system (read more at GitHub)"
```

Comment:
Installing wine-staging is never recommended for stable performance.

Reply:
Also this. I'd suggest just going with Vanilla Wine for the Deck, as staging could cause any type of issues related to its unstable updates that not everyone would appreciate. Anyways the script uses a bundled Wine build so no need to change this.